pkg/resources/validation/cluster/provisioningcluster_test.go

```diff
@@ -75,25 +75,39 @@ func Test_isValidName(t *testing.T) {
 		},
 		{
 			name:             "name length is exactly 63 characters",
-			clusterName:      "Cq8Oh6UVnTyPOITcfwrXFjJRuZ4KV2q6ItimQkcGeX1ZQGm7oa3jbLd9N0diIka",
```

Comment:
Can we add a test that ensures something like what ricky suggested isn't a valid name (like `-----------------`)

Reply:
Okay, I've added two more test cases for this scenario